draft-ietf-httpapi-idempotency-key-header.md

```diff
@@ -65,20 +65,18 @@ An idempotency key is a unique value generated by the client which the resource
 
 ## Syntax
 
-The `Idempotency-Key` request header field describes
 
+`Idempotency-Key` is an Item Structured Header {{!RFC8941}}. Its value MUST be a String (Section 3.3.3 of {{!RFC8941}}. Its ABNF is:
 
-    Idempotency-Key       = idempotency-key-value
-
-    idempotency-key-value = opaque-value
-    opaque-value          = DQUOTE *idempotencyvalue DQUOTE
-    idempotencyvalue      = %x21 / %x23-7E / obs-text
-           ; VCHAR except double quotes, plus obs-text
+    Idempotency-Key = sf-string
+    sf-string = DQUOTE *chr DQUOTE
+    chr       = unescaped / escaped
+    unescaped = %x20-21 / %x23-5B / %x5D-7E
+    escaped   = "\" ( DQUOTE / "\" )
```

Comment:
You don't need to repeat all of this - just reference `sf-string` from the sh spec

Reply:
Done.